lms/envs/common.py

```diff
@@ -1917,6 +1922,15 @@
 CERT_NAME_SHORT = "Certificate"
 CERT_NAME_LONG = "Certificate of Achievement"
 
+#################### Badgr OpenBadges generation #######################
+BADGR_API_TOKEN = None
+BADGR_BASE_URL = "http://localhost:8005"
+BADGR_ISSUER_SLUG = "example-issuer"
+# A dictionary of course modes to file paths. Example:
+# {'honor': '/path/to/badge_image.png'}
+# Badge images must be square and in PNG or SVG format.
+BADGR_IMAGE_SOURCES = {}
```

Comment:
@ottonomy Thanks for the replies!

@Kelketek See the comment about SVG support - if it isn't supported for now, we should probably remove the current mentions.